sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```
@@ -197,8 +197,16 @@ def rank_error(msg):
           # Skip empty messages.
           if m.messageImportance is None:
             continue
-          _LOGGER.info(message)
-          if str(m.messageImportance) == 'JOB_MESSAGE_ERROR':
+          message_importance = str(m.messageImportance)
```

should we have catch-all clause logged as INFO?

---

e.g. for  JOB_MESSAGE_IMPORTANCE_UNKNOWN

---

That's reasonable, done

---

Also catches any new enums that may be added (doesn't seem likely but at the very lease we'll be logging that output)